accounts/abi/bind/backends/simulated_test.go

```diff
@@ -157,7 +157,7 @@ func TestNewSimulatedBackend_AdjustTimeFail(t *testing.T) {
 	testAddr := crypto.PubkeyToAddress(testKey.PublicKey)
 	sim := simTestBackend(testAddr)
 	// Create tx and send
-	tx := types.NewTransaction(0, testAddr, big.NewInt(1000), params.TxGas, big.NewInt(1), nil)
+	tx := types.NewTransaction(0, testAddr, big.NewInt(1000), params.TxGas, big.NewInt(1), nil, nil, nil)
```

Comment:
I don't know how Typed Transaction Enveloppes are going to be handled in Geth but I am wondering if we could have convenient methods to create different types of transactions rather than setting unused parameters each time.